tools/topology/m4/bytecontrol.m4

```diff
@@ -27,6 +27,13 @@ define(`CONTROLBYTES_EXTOPS',
 `		put STR($3)'
 `	}')
 
+dnl CONTROLBYTES_EXTOPS_READONLY(info, comment, get)
```

Comment:
Can you have an inline comment here describing the control type.

Reply:
Done